src/pages/partners/affiliate-ib/index.tsx

```diff
@@ -123,47 +124,42 @@ const StyledText = styled(Text)`
 `
 
 const items: ItemsType = [
-    { title: '60K+', subtitle: <Localize translate_text="members" /> },
-    { title: '$47M+', subtitle: <Localize translate_text="paid out since inception" /> },
-    { title: '190+', subtitle: <Localize translate_text="countries" /> },
-    { title: '1M+', subtitle: <Localize translate_text="clients" /> },
+    { title: '_t_60K+_t_', subtitle: '_t_members_t_' },
+    { title: '_t_$47M+_t_', subtitle: '_t_paid out since inception_t_' },
+    { title: '_t_190+_t_', subtitle: '_t_countries_t_' },
+    { title: '_t_1M+_t_', subtitle: '_t_clients_t_' },
```

Comment:
same

Reply:
I have made the changes as suggested. Thanks